kokoro/README.md

```diff
@@ -0,0 +1,11 @@
+Kokoro Infrastructure
+----------------------
+
+The files in this directory serve as plumbing for running tests under
+Kokoro, our internal CI.
+
+We have shared this part of our CI configuration in hopes that it is
```

Comment:
I would suggest to drop this whole paragraph and replace it with:
"If there are any changes required to these config files, please file an issue."

Reply:
fixed